app/pod/components/weather-card/component.ts

```diff
@@ -2,17 +2,19 @@ import Component from '@glimmer/component';
 import weatherIcons from 'weather-app/utils/weather-icon';
 
 export default class WeatherCardComponent extends Component {
-  get icon() {
-    var prefix = 'wi wi-';
-    var code = this.args.weatherData.weather[0].id;
-    var icon = weatherIcons[code].icon;
+  get icon(): string {
+    const prefix: string = 'wi wi-';
```

Comment:
you do not need to do this here as TS will infer the type in many cases. it is something I too was doing all the time :D

Reply:
Makes sense @sirJiggles !